readme.md

```diff
@@ -38,7 +38,8 @@ choco install hyper
 3. Install the dependencies: `yarn`
 4. Build the code and watch for changes: `yarn run dev`
 5. In another terminal tab/window/pane, run the app: `yarn run app`
-
+  * If you use Visual Studio Code, use `Launch Hyper` in debugger configuration to launch a new Hyper instance with debugger attached. 
```

Comment:
I found this a little bit confusing. It should be merge with `5` line because you don't necessarily have to `yarn run app` to launch instance with VSCode.

Reply:
See new edit. I made `yarn run app` and visual studio code as options to run `hyper`.

I can also see this under another minor section on debugging, but seems an overkill for now.